src/OmniSharp.Roslyn.CSharp/Services/Types/TypeLookup.cs

```diff
@@ -38,11 +46,11 @@ public TypeLookupService(OmnisharpWorkspace workspace, FormattingOptions formatt
                     //non regular C# code semantics (interactive, script) don't allow namespaces
                     if(document.SourceCodeKind == SourceCodeKind.Regular && symbol.Kind == SymbolKind.NamedType && !symbol.ContainingNamespace.IsGlobalNamespace)
                     {
-                        response.Type = $"{symbol.ContainingNamespace.ToDisplayString()}.{symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat)}";
+                        response.Type = symbol.ToDisplayString(DefaultFormat);
                     }
                     else
                     {
-                        response.Type = symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat);
+                        response.Type = symbol.ToDisplayString(SimpleFormat);
```

Comment:
thanks for the feedback!

I think it's a great suggestion to add more comprehensive unit tests - regardless of whether we merge anything proposed here or not, the tests will at least be useful for any potential changes in the future to avoid regressions. 

I'll do that.